docs/_sass/_docs-display-component.scss

```diff
@@ -112,6 +112,12 @@
                 margin-top: -410px;
             }
         }
+
```

Comment:
can we remove this

Reply:
We can't do that. This class is used in documentation when message strip is positioned inside overlay example.